onnxruntime/contrib_ops/cpu/bert/attention.cc

```diff
@@ -328,10 +328,10 @@ Status Attention<T>::Compute(OpKernelContext* context) const {
   }
 
   // Compute the attention score and apply the score to V
-  return ApplyAttention(Q, K, V, mask_index, past, output,
-                        batch_size, sequence_length,
+  return ApplyAttention(Q, K, V, mask_index, past, output, nullptr, nullptr,
```

Comment:
it's better to add a comment for these nullptr(e.g nullptr /*present_key*/)

Reply:
Fixed